### ISSUE TITLE
Allow slide to close for videos

### ISSUE THREAD

**Issue by Beagon** [biati-digital/glightbox#286]
**Please describe the feature you want to be implemented.**
I would like to be able to close a video with sliding up or down just like an image on mobile. With an opt-in option of course.

**Explain why the feature is useful**
This would help making the experience more "mobile app like" by being able to have the same interaction on images as videos. This would also remove the need for the close button (if images and videos are the only content shown in the gallery).

**Have you seen it somewhere else?**
Most social media sites that have videos and images.

**Additional context**
I've been able to do this myself by using a custom version of gLightbox. I've added the following code to the compiled JS:

`          if (hasClass(media, 'gslide-video')) {
              mediaImage = media.querySelector('.gvideo');
          }`

On line 1126 underneath `if (hasClass(media, 'gslide-image')) {`.

This solution doesn't interfere with other types of iFrames and Inline content. I would make a full PR but my Javascript is quite rusty.

**Comment by frankie-tech** [comment]
Hey there @Beagon,

That's an interesting idea! You're more than welcome to give it a shot yourself!

Make sure to check out the contributing guide as well.

**Comment by Beagon** [comment]
@frankie-tech I'll give it a go in my free time. Would you like this to be optional or included in touch events?

**Comment by frankie-tech** [comment]
Yeah, it should be optional, the way you added the setting seems fine to me.

It should be covered by the existing `TouchEvents` class. Since you've made a PR, I'm assuming you've already figured that out, haha.
